engine/schema/migrations/20170728040000_change_table_engine_and_encoding.php

```diff
@@ -0,0 +1,173 @@
+<?php
+
+use Phinx\Db\Adapter\MysqlAdapter;
+use Phinx\Migration\AbstractMigration;
+
+class ChangeTableEngineAndEncoding extends AbstractMigration {
```

Comment:
I'm really torn about requiring this to upgrade to 3.0, though a controlled process like this is the way.